apple/internal/partials/static_framework_header_modulemap.bzl

```diff
@@ -115,14 +115,19 @@ def _create_umbrella_header(actions, output, headers):
     content = "\n".join(import_lines) + "\n"
     actions.write(output = output, content = content)
 
-def _static_framework_header_modulemap_partial_impl(ctx, hdrs, binary_objc_provider):
+def _static_framework_header_modulemap_partial_impl(ctx, hdrs, custom_umbrella_header, binary_objc_provider):
     """Implementation for the static framework headers and modulemaps partial."""
     bundle_name = bundling_support.bundle_name(ctx)
 
     bundle_files = []
 
     umbrella_header_name = None
-    if hdrs:
+    if custom_umbrella_header:
```

Comment:
I'm happy to combine this logic with the logic below, but I didn't find a great way of doing that where it was still clear to read